cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/executor/ExecutionUtilsTest.java

```diff
@@ -127,6 +129,49 @@ public void testProcessAlterPartitionReassignmentResult() throws Exception {
     EasyMock.reset(result);
   }
 
+  @Test
+  public void testGetInterBrokerReplicaTasksToReexecute() {
+    Set<TopicPartition> partitionsInMovement = Set.of(P0, P1);
+
+    ExecutionProposal proposalForPartition0 = EasyMock.mock(ExecutionProposal.class);
+    ExecutionProposal proposalForPartition1 = EasyMock.mock(ExecutionProposal.class);
+    ExecutionProposal proposalForPartition2 = EasyMock.mock(ExecutionProposal.class);
+
+    EasyMock.expect(proposalForPartition0.topicPartition()).andReturn(P0).anyTimes();
+    EasyMock.expect(proposalForPartition1.topicPartition()).andReturn(P1).anyTimes();
+    EasyMock.expect(proposalForPartition2.topicPartition()).andReturn(P2).anyTimes();
+    EasyMock.replay(proposalForPartition0);
+    EasyMock.replay(proposalForPartition1);
+    EasyMock.replay(proposalForPartition2);
+
+    ExecutionTask executionTaskForPartition0 = EasyMock.mock(ExecutionTask.class);
+    ExecutionTask executionTaskForPartition1 = EasyMock.mock(ExecutionTask.class);
+    ExecutionTask executionTaskForPartition2 = EasyMock.mock(ExecutionTask.class);
```

Comment:
Why do we need to mock this -- i.e. Can we create an actual object for each `ExecutionTask`?

Reply:
Same with above. I think those other parameters are not relevant, and not sure what values should I provide for those params.